probzelus/inference/ds_streaming_low_level.ml

```diff
@@ -37,6 +37,12 @@ let observe : type a b.
   | DSgraph_Marginalized (_, Some _) -> assert false
   end
 
+let observe_with_graft : type a b.
```

Comment:
I would use this function in `observe_conditional` a few lines below, no?

Reply:
Fixed in the commit below.